### PR TITLE
feat: update docs for jina embeddings v3

### PR DESCRIPTION
**Pull request by makram93**
This PR adds documentation for `jina-embeddings-v3`. It explains the two additional fields:

- task: to properly load adapters 
- dimensions: allow user to truncate dimension with MRL.

### CODE REVIEW

File: site/en/embeddings/embed-with-jina.md [hunk 1]
```diff
@@ -39,6 +41,20 @@ __Parameters__:
...
+  - `"retrieval.query"`: Used to encode user queries or questions in retrieval tasks.
+  - `"classification"`: Used to encode text for text classification tasks.
+  - `"text-matching"`: Used to encode text for similarity matching, such as measuring similarity between two sentences.
+  - `"clustering"`: Used for clustering or reranking tasks.
+
+- `dimensions` (*int*)
+
+  The number of dimensions the resulting output embeddings should have. Defaults to 1024. Only supported for embedding models v3 and higher. 
+
 To create embeddings for documents, use the `encode_documents()` method:
```
Comment (bwanglzu) [inline site/en/embeddings/embed-with-jina.md]: add one line describe `encode_documents` by default, use `retrieval.passage` adapter.

File: site/en/embeddings/embed-with-jina.md [hunk 2]
```diff
@@ -59,15 +75,15 @@ print("Dim:", jina_ef.dim, docs_embeddings[0].shape)
...
-        5.53560140e-01,  8.92506300e-01, -2.39408610e-01, -4.22413560e-01,
-       -3.19551350e-01,  5.59153850e-01,  2.44338100e-01, -8.60452100e-01])]
-Dim: 768 (768,)
+        2.46419664e-02,  3.45800668e-02, -1.69200692e-02, -9.93125699e-03,
+       -2.75502075e-03, -8.65739491e-03, -1.19673612e-03,  6.19480573e-03,
+       -1.12194521e-02, -3.99678349e-02,  3.19652334e-02, -1.26290624e-03])]
+Dim: 1024 (1024,)
 ```
 
 To create embeddings for queries, use the `encode_queries()` method:
```
Comment (bwanglzu) [inline site/en/embeddings/embed-with-jina.md]: add one line describes `encode_queries` will use `retrieval.passage` adapter`

**Review by bwanglzu — COMMENTED** [review]
please also add one line how ppl can encode beyond query and documents

**Review by bwanglzu — APPROVED** [review]
lgtm!

**Review by codingjaguar — APPROVED** [review]

**Pull request merged**